app/views/questions/question_content.html.erb

```diff
@@ -0,0 +1,249 @@
+                </div>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+      <div class="box-area">
+        <div class="row">
+          <div class="large-3 medium-3 small-3 columns">
+            <div class="box">
+              <a href="#" class="pen">
+                Заполните форму
+                заявки или закажите
+                обратный звонок.
+              </a>
+            </div>
+          </div>
+          <div class="large-3 medium-3 small-12 columns">
+            <div class="box">
+              <a href="#" class="girl">
+                Юрист перезвонит и
+                расскажет о этапах
+                решения проблемы.
+              </a>
+            </div>
+          </div>
+          <div class="large-3 medium-3 small-3 columns">
+            <div class="box">
+              <a href="#" class="list">
+                Поможем собрать
+                все необходимые
+                документы для дела.
+              </a>
+            </div>
+          </div>
+          <div class="large-3 medium-3 small-3 columns">
+            <div class="box">
+              <a href="#" class="flag">
+                Доведем дело
+                до логического конца.
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="comments-area">
+        <h4><%= @keyword.keyword %></h4>
+        <div class="comment-list">
+          <% @questions.each do |content| %>
+              <div class="comment">
+                <div class="ava">
+                  <img src="<%= raw content.user.avatar %>"/>
+                </div>
+                <div class="comment-body">
+                  <h4><%= content.user.firstname %> <%= content.user.lastname %></h4>
+                  <p><%= content.body %></p>
+                </div>
+                <div class="clearfix"></div>
+                <% if content.answers.present? %>
+                    <div class="comment-respond">
+                      <div class="ava">
+                        <img/>
+                      </div>
+                      <div class="comment-body">
+                        <h4><%= content["answer"] %></p></h4><span class="stars"></span>
+                        <p>
+                        <p><%= content["banswer"] %></p>
+                        </p>
+                      </div>
+                    </div>
+                    <div class="comment-link new-cursor">
+                      свернуть
+                    </div>
+                <% end %>
+              </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="page-bottom">
+  <div class="row">
+    <div class="large-12 columns">
+      <h3>
+        <span>актуальные темы:</span>
+      </h3>
+    </div>
+  </div>
+  <div class="row">
+    <div class="large-4 medium-12 small-12 columns">
+      <ul>
+        <% @urls.first(7).each do |url| %>
```

Comment:
@urs => @random_keywords

I don'like how keywords are separated here and bellow by 7. In fact we have 3 similar html blocks which could be re-used in place and build 3 different blocks with different keywords. To do this action we @random_keywords split on three parts, like:
in helper method should be placed
```
def keyword_batch(keywords)
  keywords.each_slice(3)
end
```

```

keyword_batch(random_keywords).each.do |keyword|
  <li>
    <%=
       link_to(
         url.keyword,
         controller: :questions,
         action: :question_content,
         keyword: url.translit
      )
    %>
  </li>
```